layouts/partials/textprocessing.html

```diff
@@ -96,4 +96,11 @@
   {{ $content = $content | replaceRE "blockquote class=callout" "blockquote" }}
 {{end}}
 
+{{/* Make ==text== into <mark>text</mark> */}}
+{{$mark := findRE "(?Us)==([^\\$]+)==" $content}}
```

Comment:
what does the `?Us` group do?

Reply:
U is ungreedy, s is single-line. 
It makes the regex work for all highlights instead of only the first one. 
How?
No idea.